packages/spec/src/index.ts

```diff
@@ -1,18 +1,30 @@
-export declare namespace v1 {
+/**
+ * The Standard Schema interface.
+ *
+ * Root-level alias of StandardSchemaV1.Schema.
+ */
+export type StandardSchemaV1<
```

Comment:
Should we offer both `StandardSchemaV1` and `StandardSchemaV1.Schema`? I try to reduce multiple ways when not necessary to simplify usage for users.

Reply:
I can remove the alias in the namespace if you prefer. I just thought it could be helpful to users who are looking for a `Schema` type with autocomplete on the namespace and don't know it can be used as a root-level type.

Reply:
Let's see if @colinhacks has an option here. 😆